src/main/kotlin/no/nav/tilleggsstonader/sak/journalføring/JournalpostService.kt

```diff
@@ -2,16 +2,31 @@ package no.nav.tilleggsstonader.sak.journalføring
 
 import no.nav.tilleggsstonader.kontrakter.dokarkiv.ArkiverDokumentRequest
 import no.nav.tilleggsstonader.kontrakter.dokarkiv.ArkiverDokumentResponse
+import no.nav.tilleggsstonader.kontrakter.felles.BrukerIdType
+import no.nav.tilleggsstonader.kontrakter.journalpost.Bruker
 import no.nav.tilleggsstonader.kontrakter.journalpost.Journalpost
+import no.nav.tilleggsstonader.kontrakter.journalpost.JournalposterForBrukerRequest
 import no.nav.tilleggsstonader.kontrakter.journalpost.Journalstatus
 import no.nav.tilleggsstonader.kontrakter.sak.DokumentBrevkode
 import no.nav.tilleggsstonader.kontrakter.søknad.Søknadsskjema
 import no.nav.tilleggsstonader.kontrakter.søknad.barnetilsyn.SøknadsskjemaBarnetilsyn
 import no.nav.tilleggsstonader.sak.fagsak.domain.Fagsak
+import no.nav.tilleggsstonader.sak.vedlegg.VedleggRequest
 import org.springframework.stereotype.Service
 
 @Service
 class JournalpostService(private val journalpostClient: JournalpostClient) {
+    fun finnJournalposterForBruker(personIdent: String, vedleggRequest: VedleggRequest): List<Journalpost> {
+        return journalpostClient.finnJournalposterForBruker(
+            JournalposterForBrukerRequest(
+                brukerId = Bruker(id = personIdent, type = BrukerIdType.FNR),
+                tema = vedleggRequest.tema ?: emptyList(),
+                journalposttype = listOfNotNull(vedleggRequest.journalposttype),
+                journalstatus = listOfNotNull(vedleggRequest.journalstatus),
+                antall = 10,
```

Comment:
Burde vi hente fler enn 10 her? 

Hvis vi kun henter 10 så trenger vi kanskje mulighet for å kunne hente neste 10? 
Det virker som at de har mulighet for paginering https://github.com/navikt/familie-integrasjoner/blob/main/src/main/resources/saf/saf-schema.graphql#L73-L77

Man får også info om hvor mange dokumenter som egentlige finnes 👀 
https://github.com/navikt/familie-integrasjoner/blob/main/src/main/resources/saf/saf-schema.graphql#L172-L193

Reply:
Ja, denne kan egentlig settes høyere fra start, ef har 200 men satte 10 for var usikker i starten 😅